### ISSUE TITLE
Strange mention of Guidance in Requirements

### ISSUE THREAD

**Issue by aisaac** [w3c/dxwg#428]
Hi @nicholascar 
I guess it's you who edited, this, apologies if you're not...
In the current https://w3c.github.io/dxwg/profiles/#requirements I can read:
> Guidance
This requirement implies that a profile is an identified resource that can be accessed, and hence MUST have a URI that resolves to a representation resource. This resource SHOULD be appropriate for the environment being profiled, but MAY be made available in alternative forms using negotiation.

Is it intended to be like this, with no reference to a github issue?

**Comment by akuckartz** [comment]
Additional question: Is the "MUST" refering to both "have a URI" and "resolves to a representation resource" ?

If yes: Would "have a URL" combine both?

**Comment by aisaac** [comment]
@akuckartz thanks but this issue is only about a super-basic editorial question! Questions like yours should be addressed later on, when we have clarified whether this sentence is really at the place it was intended to be (I don't think it is, as it stands out compared to what's around)

**Comment by nicholascar** [comment]
That text, in grey, was an early attempt to show how Issues (Requirements) may be addressed. I think Rob authored that one but the logic for the answer was taken from discussion.

I have created a commit (to soon be PRed) to remove this now vestigal text from the doc since we are now dealing with Issues directly in GitHub, not in the doc, until final text is agreed to.

Closing Issue with commit https://github.com/w3c/dxwg/commit/febbe0df548cd70243b1e9d75b4f37e5aa1f4a7d
